completions/drush.completion.sh

```diff
@@ -7,7 +7,7 @@
 #   http://github.com/drush-ops/drush/blob/master/drush.complete.sh
 
 # Ensure drush is available.
-which drush &> /dev/null || alias drush &> /dev/null || return
+_omb_util_command_exists drush || return
```

Comment:
`_omb_util_command_exists` => `_omb_util_binary_exists`

Reply:
This is fine as is. I overlooked `alias drush`. `_omb_util_command_exists` is used to test both the binary and alias.